.github/actions/generate-commit-list-v1/src/getRawCommitList.ts

```diff
@@ -10,15 +10,14 @@ import { GetRawCommitListParams } from './types'
 
 export default async function getRawCommitList({
   base,
-  head
+  head,
+  tag
 }: GetRawCommitListParams): Promise<string[]> {
   try {
-    const { stdout: rawCommitList } = await getExecOutput('git log', [
-      `origin/${base}..origin/${head}`,
-      '--oneline',
-      '--no-merges',
-      '--abbrev-commit'
-    ])
+    const subCommand = tag ? `${tag}..HEAD` : `origin/${base}..origin/${head}`
+    const { stdout: rawCommitList } = await getExecOutput(
```

Comment:
Should account for `stderr` errors

Reply:
@straker 
the function `getExecOutput` returns values in `stdout` if everything is ok and throws an error if not. `stderr` is never thrown. I've tested this